st2actions/st2actions/runners/mistral/v2.py

```diff
@@ -238,6 +253,51 @@ def run(self, action_parameters):
 
         return (status, partial_results, exec_context)
 
+    def resume(self, ex_ref, task_names):
+        mistral_ctx = ex_ref.context.get('mistral', dict())
+
+        if not mistral_ctx.get('execution_id'):
+            raise Exception('Unable to rerun because mistral execution_id is missing.')
+
+        execution = self._client.executions.get(mistral_ctx.get('execution_id'))
+
+        # pylint: disable=no-member
+        if execution.state not in ['ERROR']:
```

Comment:
Using a top level constant for mistral_re_runable_states or similar is probably a good idea.